src/diffpy/snmf/subroutines.py

```diff
@@ -359,10 +359,10 @@ def update_weights_matrix(
       The length of the experimental signal patterns
 
     stretching_factor_matrix: 2d array like
-      The matrx containing the stretching factors of the calculated component signals. Has dimensions K x M
+      The matrix containing the stretching factors of the calculated component signals. Has dimensions K x M
       where K is the number of component signals and M is the number of XRD/PDF patterns.
 
-    component_matrix: 2d array lik
+    component_matrix: 2d array like
```

Comment:
I believe these have been addressed in this PR: https://github.com/diffpy/diffpy.snmf/pull/121/files

could you please double check that you've synced with `main` before making any commits?

Reply:
Ok